base_import_async/models/base_import_async.py

```diff
@@ -61,8 +61,8 @@ def _create_csv_attachment(session, fields, data, options, file_name):
     # write csv
     f = StringIO()
     writer = csv.writer(f,
-                        delimiter=options.get(OPT_SEPARATOR),
-                        quotechar=options.get(OPT_QUOTING))
+                        delimiter=str(options.get(OPT_SEPARATOR)),
+                        quotechar=str(options.get(OPT_QUOTING)))
```

Comment:
@guewen note this change is part of the 8.0 forward port. So if that needs changing I'd do it in another PR.